packages/web/src/scss/helpers/_animation.scss

```diff
@@ -1,5 +1,5 @@
-$spin-duration: 1s;
-$spin-easing: linear;
+$-spin-duration: 1s;
```

Comment:
What is the difference between using `_` and `-` in variable prefixes? Does it have any meaning?

Reply:
They both make the variable private, it's just a syntactic difference coming from the [kebab-case](https://github.com/stylelint-scss/stylelint-config-standard-scss/blob/main/index.js#L51) rule which doesn't allow `_`. (I personally prefer `_` over `-` due to better readability but so far it seems I can live with `-` too.)

Reply:
Ok, thanks for enlighting me. I also prefer `_` for marking variables private. So I suggest rewriting the regex to allow `_`. What do you think?

Reply:
OK and do we actually want to keep `_` allowed only or to keep both chars available? @adamkudrna @literat

Reply:
As we agreed, only `_` will be allowed.

Reply:
Waiting for merge: https://github.com/lmc-eu/code-quality-tools/pull/106